### PR TITLE
chore: add image embedding test

### PR DESCRIPTION
**Pull request by maxnachlinger**
Added this silly test to learn how to embed pngs in the svgs this library produces :)

### CODE REVIEW

File: src/tests/svg.rs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+use base64::{engine::general_purpose, Engine as _};
+use crate::{ECL, QRBuilder};
+use crate::convert::Builder;
+use crate::convert::svg::SvgBuilder;
+
+#[test]
+fn it_embeds_an_img_src() {
+    let image_bytes = std::fs::read("assets/red_square.png").unwrap();
+    let image_base64 = general_purpose::STANDARD.encode(&image_bytes);
+    let image_src = format!("data:image/png;base64,{image_base64}");
```
Comment (maxnachlinger) [inline src/tests/svg.rs:10]: This data URI embeds the image directly into the SVG, not just as a link.

File: src/tests/svg.rs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+use crate::convert::Builder;
+use crate::{QRBuilder, ECL};
+use base64::{engine::general_purpose, Engine as _};
+
+#[cfg(feature = "svg")]
+#[test]
+fn it_embeds_an_image_via_data_uri() {
+    let image_bytes = std::fs::read("assets/red_square.png").unwrap();
+    let image_base64 = general_purpose::STANDARD.encode(&image_bytes);
+    let data_uri = format!("data:image/png;base64,{image_base64}");
```
Comment (maxnachlinger) [inline src/tests/svg.rs:12]: This data URI embeds the image directly into the SVG, not just as a link. Let me know if you'd like me to add an example for this as well.

Comment (erwanvivien) [reply src/tests/svg.rs:12]: I think it's good too, do you think we should add the file asset & the dev dependency to test that?
Maybe a simple url / inline base64 could do the trick? :)

Comment (maxnachlinger) [reply src/tests/svg.rs:12]: Sure, I'll modify this to use a test base64 string and we can drop the extra dependency

**Comment by erwanvivien** [comment]
Thanks a lot max 👍

**Pull request merged**